taichi/struct/struct_llvm.cpp

```diff
@@ -167,9 +171,11 @@ void StructCompilerLLVM::generate_types(SNode &snode) {
 
   // Create a dummy function in the module with the type stub as return type
   // so that the type is referenced in the module
+  std::string func_name = type_stub_name(&snode) + "_func";
```

Comment:
can we wrap these lines into a function so that it's easier to update in the future?